rooms.js

```diff
@@ -110,6 +110,7 @@ class BasicRoom {
 		this.privacySetter = null;
 		/** @type {Map<string, ChatRoom>?} */
 		this.subRooms = null;
+		this.gameNumber = 0;
```

Comment:
:|

Reply:
Why is this ':|'. I added this here because multiple roomgames might use the gamenumber field. (I had it planned that way at least).